kmod/scoutfs-kmod.spec.in

```diff
@@ -111,8 +117,8 @@ SCOUTFS_RPM_NAME=$(rpm -q %{name} | grep "%{version}-%{release}")
 rpm -ql $SCOUTFS_RPM_NAME | grep '\.ko$' > /var/run/%{name}-modules-%{version}-%{release} || true
 
 %postun
-if [ -x /sbin/weak-modules ]; then
-    cat /var/run/%{name}-modules-%{version}-%{release} | /sbin/weak-modules --remove-modules --no-initramfs
```

Comment:
Like up in the %post section, just drop the absolute path and let the command be found in the PATH.

Reply:
Sounds good, I thought it was bizarre that the original spec did things differently but had no clue if there were legitimate concerns about PATH being weird on certain platforms (minor versions of RHEL, different EL-alikes, who knows).